code-injection-analysis/MiltonFSiqueira_Folha-de-ponto-RC-andaimes/script.js

```
@@ -0,0 +1,207 @@
+    console.log(data);
+  } catch (err) {
+    console.error(err);
+  }
+} 
+
+//Cadastro e Autenticação de Usuários
+
+// auth.js
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcrypt');
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)